state/action.go

```diff
@@ -49,6 +50,13 @@ const (
 
 	// ActionRunning indicates that the Action is currently running.
 	ActionRunning ActionStatus = "running"
+
+	// ActionAborting indicates that the Action is running but should be
+	// aborted.
+	ActionAborting ActionStatus = "aborting"
```

Comment:
can we add these extra status values to the CLI list-operations command checks

Reply:
yes but so you know @wallyworld missed ActionCancelled from list-operations :grinning: